project/Common.scala

```diff
@@ -62,6 +63,8 @@ object Common extends AutoPlugin {
         "-skip-packages",
         "akka.pattern" // for some reason Scaladoc creates this
       ),
+    // FIXME enable "-Xfatal-warnings", but then we need the silencer
```

Comment:
In Scala 2.13.2 `@nowarn` is built-in.

Reply:
created issue https://github.com/akka/akka-projection/issues/108